assets/versions/versions.json

```diff
@@ -0,0 +1,4 @@
+{
+  "cliVersion": "v0.5.0",
+  "configFileSchemaVersion": "v1"
+}
```

Comment:
@zaremba-tomasz just getting back to this, what's the reason to keep this in a json file and not in code?